### PR TITLE
Awesome books es6

### PR DESCRIPTION
**Pull request by karam084**
* Project requirements

- Create a new directory and initialize it with git.
- Use npm init -y command to create package.json file.
- Create the entry point for your JavaScript code called index.js
- Create index.css file for your styles.
- Create the main index.html file. Link your index.js and index.css files there.
- Create a directory called modules.
- Re-create your Awesome books app following the rules
- Use ES6 syntax.
- Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file. For this exercise - the more modules the better!
- Keep all functionalities of the app without errors.
- Display the current date below the navigation bar. - For this requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+const btnAdd = document.getElementById('Add');
+const showList = document.getElementById('showList');
+const showBook = document.getElementById('bookList');
+const addBook = document.getElementById('addBook');
+const contact = document.getElementById('contact');
+const showAdd = document.getElementById('showAdd');
+const showCon = document.getElementById('showCon');
+const time = document.getElementById('time');
+
+function showBooks() {
```
Comment (Hamzaoutdoors) [inline index.js]: - [x] Kindly use `arrow` syntax for functions for best practice of **JavaScript ES6** 👍🏼

File: index.js [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    showAdd.classList.remove('colBlue');
+  }
+  if (contact.classList.contains('visible')) {
+    contact.classList.remove('visible');
+    contact.classList.add('Invisible');
+    showAdd.classList.remove('colBlue');
+  }
+}
+
+function AddShow() {
```
Comment (Hamzaoutdoors) [inline index.js]: - [x] Kindly use `arrow` syntax for functions for best practice of **JavaScript ES6** 👍🏼

File: index.js [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    showList.classList.remove('colBlue');
+  }
+  if (contact.classList.contains('visible')) {
+    contact.classList.remove('visible');
+    contact.classList.add('Invisible');
+    showCon.classList.remove('colBlue');
+  }
+}
+
+function showContact() {
```
Comment (Hamzaoutdoors) [inline index.js]: - [x] Kindly use `arrow` syntax for functions for best practice of **JavaScript ES6** 👍🏼

File: index.js [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+    showList.classList.remove('colBlue');
+  }
+  if (addBook.classList.contains('visible')) {
+    addBook.classList.remove('visible');
+    addBook.classList.add('Invisible');
+    showAdd.classList.remove('colBlue');
+  }
+}
+
+function Add() {
```
Comment (Hamzaoutdoors) [inline index.js]: - [ ] Kindly use `arrow` syntax for functions for best practice of **JavaScript ES6** 👍🏼

File: index.js [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+  if (bookTitle.value !== '' && bookAuthor.value !== '') {
+    bList.AddBook(bookTitle.value, bookAuthor.value);
+    localStorage.setItem('BookList', JSON.stringify(bList.Books));
+    window.location.reload();
+  }
+}
+
+function Remove() {
```
Comment (Hamzaoutdoors) [inline index.js]: - [x] Kindly use `arrow` syntax for functions for best practice of **JavaScript ES6** 👍🏼

File: index.js [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+  if (this.id > -1) {
+    bList.RemoveBook(this.id);
+    localStorage.setItem('BookList', JSON.stringify(bList.Books));
+    window.location.reload();
+  }
+}
+
+function displayBooks() {
```
Comment (Hamzaoutdoors) [inline index.js]: - [x] Kindly use `arrow` syntax for functions for best practice of **JavaScript ES6** 👍🏼

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @karam084, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

- [x] Pleas add `/node_modules` in .gitignore file

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Please make sure you add meaningful messages to your commits, `Add modules` is very generic and if you find a bug you won't be able to trace down what commit caused it because all the commit messages are the same. Keep this in mind for your future projects as it is a good practice. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @hamzaoutdoors in your question so I can receive the notification ^_^

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi  @karam084 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**